<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeEditDialog</class>
 <widget class="QDialog" name="EmployeeEditDialog">
  <property name="minimumWidth">
   <number>360</number>
  </property>
  <property name="windowTitle">
   <string>Сотрудник</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing"><number>8</number></property>
     <property name="horizontalSpacing"><number>12</number></property>
     <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
     <property name="labelAlignment"><set>Qt::AlignRight|Qt::AlignVCenter</set></property>
     <item row="0" column="0">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lastNameEdit">
       <property name="placeholderText">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="firstNameEdit">
       <property name="placeholderText">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="middleNameLabel">
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="middleNameEdit">
       <property name="placeholderText">
        <string>Отчество (необязательно)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="positionLabel">
       <property name="text">
        <string>Должность:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="positionCombo">
       <property name="minimumWidth">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="userLabel">
       <property name="text">
        <string>Пользователь:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="userCombo">
       <property name="minimumWidth">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="badgeLabel">
       <property name="text">
        <string>Штрихкод бейджа:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="badgeEdit">
       <property name="placeholderText">
        <string>Для входа по сканированию (необязательно)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
